docs/quickstart.rst

```diff
@@ -1,4 +1,166 @@
 .. _quickstart:
 
 Quickstart
-==========
```

Comment:
This is good to go as is for now. There are a couple of tweaks I'll do in a future PR. Most notably, I'd like to call all the metrics that will be later shown in the visualization (e.g., accuracy_score_group_summary, demographic_disparity_difference and accuracy_score_difference).